app/controllers/concerns/current_project.rb

```diff
@@ -15,6 +15,7 @@ def current_project
 
   def require_project
     @project = (Project.find_by_param!(params[:project_id]) if params[:project_id])
+    @project ||= (Project.find_by_param!(params[:id]) if params[:id])
```

Comment:
I think loading the project by id as a fallback might be dangerous and lead us to load the wrong project. In a URL using both `project_id` and `id` as path variables, we wouldn't want this to happen.

In another similar situation, we overrided the `require_project` method to match the specific need. See `ProjectsController` for an example of what I mean.